SMTPServerUnitTests/pch.h

```diff
@@ -0,0 +1,12 @@
+// pch.h: This is a precompiled header file.
```

Comment:
Add here message for PSV-studio free version, so Jenkins can build your project.